src/model/document.ts

```diff
@@ -97,6 +97,14 @@ export interface FeatureKinds {
   'js-import': Import;
 }
 
+export type QueryOptions = object & {
```

Comment:
this reads a little funny vs an interface. It's a shame an interface can't extend `object`, and that non-objects satisfy the implicit interface of empty classes.